projects/cdk/directives/auto-focus/autofocus.directive.ts

```diff
@@ -85,27 +88,21 @@ export class TuiAutoFocusDirective implements AfterViewInit {
     }
 
     private iosWebkitAutofocus() {
-        const fakeInput: HTMLElement = this.renderer.createElement('input');
+        const fakeInput: HTMLElement = this.createFakeInputIos();
+        const duration = this.getMinAnimationTimeBeforeFocus();
 
-        fakeInput.style.position = 'absolute';
-        fakeInput.style.opacity = '0';
-        fakeInput.style.height = '0';
-
-        const blurHandler = () => setNativeFocused(fakeInput);
+        const blurHandler = () => setNativeFocused(fakeInput, true, true);
```

Comment:
**nit**

let' make
```
const fakeInput: HTMLInputElement
```

and replace  
```
setNativeFocused(fakeInput, true, true)
```
by 
```
fakeInput.focus({preventScroll: true});
```

I think it is more readable than arguments `true, true`

Reply:
fixed